doc/irmin-pack/design/lower_layer.md

```diff
@@ -150,7 +150,7 @@ If a store is opened with a `lower_layer` configured and its upper layer control
 1. Create empty `store.1.suffix` in upper layer
 1. Create empty prefix and mapping in upper layer
 1. Update upper layer control file
-    - `status` is `Gced`
+    - `status` is GC'd
```

Comment:
```suggestion
    - `status` is `Gced`
```
Sorry this specific `Gced` actually refers to a term in the code so we can't change it :P

Reply:
Oh! Good to know!